js/src/components/contentAnalysis/ReadabilityAnalysis.js

```diff
@@ -46,10 +47,11 @@ ReadabilityAnalysis.propTypes = {
  * @returns {Object} Props that should be passed to ContentAnalysis.
  */
 function mapStateToProps( state, ownProps ) {
-	let marksButtonStatus = state.marksButtonStatus;
-
+	let marksButtonStatus;
 	if ( ownProps.hideMarksButtons ) {
 		marksButtonStatus = "hidden";
+	} else {
+		marksButtonStatus = "enabled";
```

Comment:
Not using the state here anymore, would that not be causing problems?

Reply:
Having 'enabled' as default, and overriding this if the state.hideMarksButtons is set seems like a more logical approach.